include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -481,4 +481,13 @@ SECTIONS
 	KEEP(*(.gnu.attributes))
 	}
 
+/* This section must be at the end of the linker script */
```

Comment:
Perhaps:     /* Must be liastin romable region */
This is similar to 

    /* Must be first in ramable region */